modules/virt-creating-storage-class.adoc

```diff
@@ -41,21 +41,22 @@ metadata:
 provisioner: kubevirt.io/hostpath-provisioner
 reclaimPolicy: Delete <2>
 volumeBindingMode: WaitForFirstConsumer <3>
+parameters:
+ storagePool: local <4>
```

Comment:
We can use storagePool only for CSI Storage class with 
provisioner: **kubevirt.io.hostpath-provisioner**

Reply:
Noted.